packages/admin/src/form/SubmitResult.tsx

```diff
@@ -0,0 +1,17 @@
+interface SubmitErrorProps<TSubmitError = unknown> {
```

Comment:
I would not call this interface `...Props`, as Props is brought into connection with React components. I'd remove the interface all together and pass the params directly to the constructor.